spec/presenters/two_factor_authentication/set_up_webauthn_selection_presenter_spec.rb

```diff
@@ -17,19 +17,6 @@
     end
   end
 
-  describe '#render_in' do
```

Comment:
I think we lost test coverage here, we still have a `SetUpWebauthnSelectionPresenter#render_in` method.